.puppet-lint.rc

```diff
@@ -0,0 +1 @@
+
```

Comment:
let's put some real config here. we can maybe take some of our own for here

Reply:
https://tickets.puppetlabs.com/browse/PDK-1202?focusedCommentId=614776&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-614776

Doesn't work. I've added it to .sync.yml and it will work eventually

Reply:
https://tickets.puppetlabs.com/browse/PDK-1246